.dockerignore

```diff
@@ -0,0 +1,31 @@
+# This file is part of REANA.
+# Copyright (C) 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
+.git
```

Comment:
We discussed keeping the `git` in the container for now, as it may help with debugging of "shared" containers while we are in the alpha stage.

Also, `reana-client` does not really need a dockerignore file, since we are not building the Docker image here.
Basically only the repositories with `Dockerfile` would need those.

Reply:
Ah yes right!